source/event_loop.c

```diff
@@ -244,6 +244,12 @@ void aws_event_loop_schedule_task_future(
     event_loop->vtable->schedule_task_future(event_loop, task, run_at_nanos);
 }
 
+void aws_event_loop_cancel_task(struct aws_event_loop *event_loop, struct aws_task *task) {
```

Comment:
Documentation says this must be called from event_loop's thread, should we assert on that here?